app/views/homepage/index.html.erb

```diff
@@ -13,7 +17,11 @@
   <%
     # The GA4 tracking on the homepage depends on some hardcoded values for index. If a section is added or removed, these values will need to be updated.
     # Specifically 'index_section_count' (the total number of sections) and 'index_section' (the index of the section e.g. 2 for the 2nd section).
-    index_section_count = 6
+    if new_design?
+      index_section_count = 5
+    else
+      index_section_count = 6
+    end
```

Comment:
I think you could also write this as

```Ruby
index_section_count = 6
index_section_count = 5 if new_design?
```

Reply:
Thanks Andy, this looks much cleaner, I've updated the view template to use this approach 👍